app/serializers/api/v1/message_serializer.rb

```diff
@@ -3,14 +3,10 @@ class MessageSerializer < ApplicationSerializer
     embed :ids, include: true
 
     attributes :id, :body, :state, :is_private, :created_at,
-      :updated_at, :offer_id, :item_id, :designation_id, :order_id
+      :updated_at, :messageable_type, :messageable_id
```

Comment:
This change really pleases me! It's the whole reason for switching to polymorphism